dair_pll/drake_utils.py

```diff
@@ -251,15 +268,18 @@ def __init__(self,
         # to False, in the hopes of saving computation time; may cause
         # re-initialization to produce erroneous visualizations.
         visualizer = None
-        if enable_visualizer:
-            visualizer = VideoWriter.AddToBuilder(
-                filename=file_utils.get_experiment_video_filename(),
-                builder=builder, sensor_pose=SENSOR_POSE, fps=FPS,
-                width=VIDEO_PIXELS[1], height=VIDEO_PIXELS[0], fov_y=CAM_FOV)
+        if visualization_file:
+            visualizer = VideoWriter.AddToBuilder(filename=visualization_file,
+                                                  builder=builder,
+                                                  sensor_pose=SENSOR_POSE,
+                                                  fps=FPS,
+                                                  width=VIDEO_PIXELS[1],
+                                                  height=VIDEO_PIXELS[0],
+                                                  fov_y=CAM_FOV)
 
         # Adds ground plane at ``z = 0``
-        halfspace_transform = RigidTransform()
-        friction = CoulombFriction(1.0, 1.0)
+        halfspace_transform = RigidTransform_[float]()
+        friction = CoulombFriction_[float](1.0, 1.0)
```

Comment:
Just out of curiosity, why these changes from `RigidTransform, CoulombFriction` to `RigidTransform_[float], CoulombFriction_[float]`?

Reply:
Something in the updates to `mypy` since 3.8 caused these lines to fail static type checking. `RigidTransform` is an alias for `RigidTransform_[float]` so the functionality is the same.